solana/pyth2wormhole/program/src/attest.rs

```diff
@@ -143,9 +252,9 @@ pub fn attest(ctx: &ExecutionContext, accs: &mut Attest, data: AttestData) -> So
     let post_message_data = (
         bridge::instruction::Instruction::PostMessage,
         PostMessageData {
-            nonce: data.nonce,
-            payload: price_attestation.serialize(),
-            consistency_level: data.consistency_level,
+	    nonce: 0, // Superseded by the sequence number
```

Comment:
Is it the case that attest will be executed more than once in a slot? If so you will still need different nonces if two attests are done in the same block, the way this is done in Wormhole is to just make the caller pass `nonce` in the instruction data and forward it through. Likely not a problem here though as far as I can tell.

Reply:
I don't think two of them would end up in the same slot. On a different note, I believe at one point you suggested that seqno has exactly the same job and that nonce is a relic+ of the past?

Reply:
It's a relic in that sequence and nonce perform the same job but it's still hashed by guardians in the VAA payload, and `node` still de-duplicates within a block for the same reason, so we still have to work with it.

Reply:
Example for the token_bridge in the JS SDK: https://github.com/certusone/wormhole/blob/dev.v2/sdk/js/src/token_bridge/attest.ts#L44 -- we can just let the user specify it when calling P2W and let them fill it with a random number.

Reply:
So the guardians don't use the sequence number when hashing? Why?

Reply:
**Are we concerned about duplicate seqnos for two separate messages processed in the same slot?**
I just confirmed that the seqno *is* used for hashes that later are used for signatures, as seen here: https://github.com/certusone/wormhole/blob/dev.v2/node/pkg/vaa/structs.go#L324

I'm sorry, but I'm struggling to understand the importance of the seqno, because as it stands, the seqno is just as good if not better.